src/client/data/AdminApi.ts

```diff
@@ -187,6 +187,32 @@ export const registerNewEvent = (event: NewEventFormData) => {
   );
 };
 
+export const editExistingEvent = (id: string, event: Partial<EventFormData>) => {
+  const { logo, closeTimeDay, closeTimeMonth, closeTimeYear, ...eventWithoutFields } = event;
+  const closeTime: string = moment(new Date(
+    closeTimeYear,
+    closeTimeMonth - 1,
```

Comment:
This function is modeled after the `registerNewEvent()` function just about it - where `closeTimeMonth - 1` is also passed. 
https://github.com/UCSDTESC/Check-in/blob/2004f7a2261007b60956a01a39f8d5731d8b2043/src/client/data/AdminApi.ts#L169-L188
Since this function gets the form data directly (where we use base 1 for months), I think it makes sense to keep it the way it is. Feel free to disagree.

Reply:
How much work would it be to reduce the month selector to base 0? It seems confusing where we need to shift the month.

Reply:
Not hard at all. We shift to base 1 when talking to the server. I'll switch it over now.